From fe0e9025cd7de41c0bb2dcc092272afbb7429c8a Mon Sep 17 00:00:00 2001
From: zoltanvb <balogh.zoltan.v@gmail.com>
Date: Fri, 1 Dec 2023 20:48:42 +0100
Subject: [PATCH] Fix videocore + switchres compile failure

After commit f24893b, some definitions were not included when RA
is compiled with --enable-videocore option (RPi legacy driver).
---
 gfx/video_crt_switch.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/gfx/video_crt_switch.c b/gfx/video_crt_switch.c
index f4706c839608..ca8b8ca392c5 100644
--- a/gfx/video_crt_switch.c
+++ b/gfx/video_crt_switch.c
@@ -33,10 +33,8 @@
 #include "../paths.h"
 #include "gfx_display.h"
 
-#if !defined(HAVE_VIDEOCORE)
 #include "../deps/switchres/switchres_wrapper.h"
 static sr_mode srm;
-#endif
 
 #ifdef HAVE_CONFIG_H
 #include "../config.h"
